wren-ui/src/apollo/server/managers/dataSourceSchemaDetector.ts

```diff
@@ -100,8 +101,171 @@ export default class DataSourceSchemaDetector
       );
     }
 
+    const getAffectedCalculatedFieldsAndRelationships = async (
```

Comment:
Could we separate this function to a new member function instead ?

Reply:
Done by c9d4b7b9